src/main/java/co/omise/requests/RequestBuilder.java

```diff
@@ -22,12 +24,17 @@
     private final Serializer serializer = Serializer.defaultSerializer();
     private static final MediaType JSON_MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
 
+    protected static final String POST = "POST";
```

Comment:
If I'm being honest, I don't see any immediate benefit of using enum here. Please share any thoughts you have on this

Reply:
Becuase `String` still can be any string. But for enum it more safety, you can force them to return enum type of `METHOD`.

Reply:
Yes I understand what enums do :) But for something this simple, I feel it's overkill. Also, if users want to implement their own Request with a method we don't have, then they are out of luck.

Reply:
I think overkill, since Java doesn't have native string enums as well.

Reply:
🤔 my initial thought is
`if users want to implement their own Request with a method we don't have` but this will call our API which means that there's no point for the users to have a custom request method? Since we would reject those requests anyway.

`I feel it's overkill` How's overkill do you think it is?

Reply:
Java doesn't have native string enums so you'll need to implement a 40-50 LoC just to avoid string typos. So I don't think it's worth it. Method names aren't that sensitive to typos anyway.

I think we shouldn't expose actual method directly to merchants, too. When they call `createCharageRequest` for example, it's already set to a POST request without needing to supply any arg right?